services/collectionService.ts

```diff
@@ -55,13 +55,15 @@ export async function collectionService({ params }: CollectionParamsProps) {
 
     const { name, symbol } = collection
 
-    const seo = await buildCollectionSEO(name, symbol)
+    const seo = buildCollectionSEO(name, symbol)
 
     return {
       props: {
-        contractAddress: tokenAddress,
-        collection,
-        seo,
+        fallback: {
```

Comment:
Maybe initialCollection rather than fallback?

Reply:
Next use this naming in their docs and I like that it shorter then any `initialData` notation